WinUIGallery/ControlPages/LinePage.xaml

```diff
@@ -0,0 +1,204 @@
+&lt;Polyline Stroke="Black" StrokeThickness="$(Slider1)"
+Points="10,100 60,40 200,40 250,100"/&gt;
+                </x:String>
+
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Slider1" Value="{x:Bind polyLineSlider1.Value, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+
+        </local:ControlExample>
+
+        <!-- PATH SAMPLE -->
+        <local:ControlExample x:Name="Example6" HeaderText="Path" >
+       
+            <Canvas Width="320" Height="200">
+                <StackPanel>
+                    <TextBlock>
+                   Draws a series of connected lines and curves.
+                    </TextBlock>
+                    <Path Stroke="DarkGoldenRod" 
+      StrokeThickness="{x:Bind pathSlider1.Value, Mode=TwoWay}"
+      Data="M 10,100 C 100,25 300,250 400,75 H 200" />
+                </StackPanel>
+                
+                <TextBlock Text="Point #1: (10,100)" Canvas.ZIndex="1" Canvas.Left="0" Canvas.Top="130" Visibility="{x:Bind ToggleSwitch.IsOn,Mode=OneWay}"/>
+                <TextBlock Text="Point #2: (100,25)" Canvas.ZIndex="1" Canvas.Left="40" Canvas.Top="75" Visibility="{x:Bind ToggleSwitch.IsOn, Mode=OneWay}"/>
+                <TextBlock Text="Point #3: (300,250)" Canvas.ZIndex="1" Canvas.Left="280" Canvas.Top="175" Visibility="{x:Bind ToggleSwitch.IsOn, Mode=OneWay}"/>
+                <TextBlock Text="Point #4: (400,75)" Canvas.ZIndex="1" Canvas.Left="360" Canvas.Top="60" Visibility="{x:Bind ToggleSwitch.IsOn, Mode=OneWay}"/>
+                <TextBlock Text="Point #5: (200,75)" Canvas.ZIndex="1" Canvas.Left="170" Canvas.Top="60" Visibility="{x:Bind ToggleSwitch.IsOn, Mode=OneWay}"/>
+            </Canvas>
+           
+            <local:ControlExample.Options>
+                <StackPanel Width="220">
+                    <ToggleSwitch x:Name="ToggleSwitch" Header="Show points" IsOn="False" />
+                    <Slider x:Name="pathSlider1" Header="Stroke Thickness" Minimum="2" Maximum="10"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Path Stroke="DarkGoldenRod" StrokeThickness="$(Slider1)"
+Data="M 10,100 C 100,25 300,250 400,75 H 200"/&gt;
+
+&lt;!-- The first segment is a cubic Bezier curve that begins at Point #1 and ends at Point #4, which is drawn by using Point #2 and 3 as the two control points. This segment is indicated by the "C" command in the Data attribute string. --&gt;
+&lt;!-- The second segment begins with an absolute horizontal line command "H", which specifies a line drawn from the preceding subpath endpoint (Point #4) to a new endpoint (Point #5). Because it's a horizontal line command, the value specified is an x-coordinate. --&gt;
```

Comment:
How about putting this above the control? I think most people would expect comments to be before the code they are documenting.

Reply:
Done!